docker/run_ditck.sh

```diff
@@ -40,26 +40,35 @@ ant -version
 REPORT=${WORKSPACE}/330tck-report
 mkdir -p ${REPORT}
 if [ -z "${JAVAX_INJECT_TCK_URL}" ];then
-  JAVAX_INJECT_TCK_URL=https://github.com/javax-inject/javax-inject/releases/download/1/javax.inject-tck.zip
+  JAVAX_INJECT_TCK_URL=http://download.eclipse.org/ee4j/cdi/jakarta.inject-tck-1.0-bin.zip
 fi
 if [ -z "${JSR299_TCK_URL}" ];then
-  JSR299_TCK_URL=https://sourceforge.net/projects/jboss/files/CDI-TCK/1.0.6.Final/jsr299-tck-1.0.6.Final-dist.zip/download
+  JSR299_TCK_URL=http://download.eclipse.org/ee4j/cdi/cdi-tck-2.0.6-dist.zip
 fi
 
 wget ${JAVAX_INJECT_TCK_URL} -O ${WORKSPACE}/javax.inject-tck.zip 
 wget ${JSR299_TCK_URL} -O ${WORKSPACE}/jsr299-tck.zip
 unzip ${WORKSPACE}/javax.inject-tck.zip  -d ${WORKSPACE}
 unzip ${WORKSPACE}/jsr299-tck.zip -d ${WORKSPACE}
 
+# Install the porting lib
+cd ${WORKSPACE}/cdi-tck-2.0.6/weld/porting-package-lib
+mvn install
+echo "+++ Installed CDI TCK porting libs"
+ls target/dependency
+cd ${WORKSPACE}
+
 #Edit test properties
 sed -i "s#ts.home=.*#ts.home=${WORKSPACE}#g" ${TS_HOME}/build.properties
-sed -i "s#porting.home=.*#porting.home=${TS_HOME}#g" ${TS_HOME}/build.properties
+sed -i "s#porting.home=.*#porting.home=${WORKSPACE}/cdi-tck-2.0.6/weld/porting-package-lib/target/dependency#g" ${TS_HOME}/build.properties
```

Comment:
From the comment description in build.properties porting.home should be TS_HOME(=${WORKSPACE}/330-tck-glassfish-porting) I presume.